src/main/java/org/frc5687/powerup/robot/OI.java

```diff
@@ -204,4 +208,20 @@ public void initializeButtons(Robot robot) {
         DriverStation.reportError("arm " + (robot.getArm()==null), false);
     }
 
+    public void poll() {
```

Comment:
I just really don't like this. You end up sending two calls per gamepad per tick, whether we're in endgame or not. Instead, you should do something like the KillAll command where it checks if there are 30 to 28 seconds left in the match, and then launches a RumbleForNSeconds command, so you would only make two calls per gamepad at the start and another two per gamepad at the end.